WcaOnRails/app/views/layouts/application.html.erb

```diff
@@ -15,11 +15,6 @@
   <%= stylesheet_pack_tag 'application' %>
   <%= javascript_pack_tag 'application' %>
 
-  <% if content_for :include_gmaps %>
-    <script type="text/javascript" src='//maps.google.com/maps/api/js?libraries=places&callback=wca._onGoogleMapsLoaded&key=<%= ENVied.GOOGLE_MAPS_API_KEY %>'></script>
```

Comment:
It doesn't have to happen in this PR, but we should remember to remove this environment variable from our chef scripts, just to keep things clean.